starboard/shared/starboard/media/iamf_util.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_SHARED_STARBOARD_MEDIA_IAMF_UTIL_H_
+#define STARBOARD_SHARED_STARBOARD_MEDIA_IAMF_UTIL_H_
+
+#include <limits>
+#include <string>
+
+namespace starboard {
+namespace shared {
+namespace starboard {
+namespace media {
+
+enum IamfSubstreamCodec {
+  kIamfSubstreamCodecOpus,
+  kIamfSubstreamCodecMp4a,
+  kIamfSubstreamCodecFlac,
+  kIamfSubstreamCodecIpcm,
+  kIamfSubstreamCodecUnknown
```

Comment:
Unless this is a map of spec defined somewhere, consider moving kIamfSubstreamCodecUnknown to the first, so when the value is zero initialized, it's initialized to kIamfSubstreamCodecUnknown.